src/content/self-hosted/install/releases.mdx

```diff
@@ -6,17 +6,58 @@ id: releases
-
-### Features
-- Add support for Kubernetes [1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md)
-- Support for custom [labels and annotations](self-hosted/administration/configuration.mdx#namespace) in ingresses managed by Okteto.
+10 May, 2023
 
 ### Breaking changes
+
 - The default value for the buildkit service session affinity is now `None`. To keep using `ClientIP` as `sessionAffinity` you need to set this value in your [buildkit configuration](self-hosted/administration/configuration.mdx#buildkit).
 
 ### Deprecation Notice
-- Support for Kubernetes [1.22](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md) has been deprecated and will be removed next release.
+
+- Support for Kubernetes [1.22](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md) has been deprecated and will be removed next release (1.9).
+
+### Features
+
+- Add support for Kubernetes [1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md)
+- Support for custom [labels and annotations](self-hosted/administration/configuration.mdx#namespace) in ingresses managed by Okteto
+- Upgraded the Okteto CLI version to [2.15.2](https://github.com/okteto/okteto/releases/tag/2.15.2)
+  - Okteto CLI 2.15.2 enables the optimisation removed at Okteto CLI 2.14.3. Now we calculate a hash for a given commit with the info around the build.
+- Self-hosted will now show an installation setup guide by default
+- Always use the baggage header for istio divert driver
+- Remove the `email` field from the helm values file. It is is now provided by the license
+- PSA Okteto Admission Webhook for restrictive PSP emulation
+- Divert virtual service transformation is now done at the mutation webhook by adding the annotation to the virtualservices
+- The frontend app now uses proper browser history routing instead of hash routing
+- Catalog items are now sorted in the same way in both the admin dashboard and a developer's deploy/redeploy modal
+- External resources can now be destroyed from the Okteto UI
+- External resources now supports [additional icons](reference/manifest.mdx##icon-string-optional)
+
+### Improvements
+
+- Added service in the mutation webhook to add the `dev.okteto.com/namespace` label to all external resources created or updated
+- Improved the performance of the query for retrieving endpoints for a preview environment using the `dev.okteto.com/namespace` label
+
+### Bugfixes
+
+- When a namespace is going to sleeping state, ingress class for ingresses at that namespace will only change under activation of unsupported flag
+- Custom endpoints annotation `dev.okteto.com/endpoints` is now mapped with the correct service of an ingress
+- Using personal repositories from Bitbucket is now possible as we've addressed an issue with restricted characters in repository names
+- Update proper link to external storage documentation in the installation dashboard
```

Comment:
It's the first time we ship the installation wizard, no need to document previous bugs